taas/__init__.py

```diff
@@ -141,15 +144,36 @@ def normalise_sheet(raw, mapping):
 
         Raw is the data as produced from read_sheet_csv().
         Mapping is a dict of cooked->raw pairs (eg: 'label' : 'Preferred Term')
+
+        Fields containing a . in their name will be turned into maps. Eg:
+        `label.en`, `label.es` and `label.fr` become three keys in the `label` map.
+        Only one level of embedding is supported.
```

Comment:
@emmajane: I'm fond of "as of (date)" in wikipedia, but I'm not fond of it in method documentation.

"As of (date)" is much suited to writing documentation where there is an external locus of control. "As of March 25th, the Prime Minister of Australia is Malcolm Turnbull". As the person writing the documentation, I don't get to change who is Prime Minister, and the process of changing the PM is unlikely to also update our docs. Since it's likely they'll get out of sync at some point, "as of..." makes sense in this context.

But when we're dealing with method documentation, the code is *right there*. Whoever is updating the code can *also* update the docs, and in fact they *should* be updating the docs. Changing the code without updating the documentation to match is poor process.

If our code had a restriction due to an external library, then "as of..." would make sense. But writing it here feels like we're excusing our future selves from keeping the documentation up-to-date.

The need to update the code is documented in the code itself, which has `TODO` comments marking where changes can be made to support greater levels of nesting, which is better than in the method docs as they show precisely where changes must be made. They show up in red on my editor with default settings when viewing the code. They show up in the code-climate [list of defects](https://codeclimate.com/github/UN-OCHA/taas/issues). If we could use github for issues, then codeclimate could automatically flip those into issues for us, and it would be both easier to see what limitations we have outstanding, and also to solicit contributions for their improvement.

I could update the docs to say "only one level of embedding is supported at this time", but I'm not sure of the value of doing so. It doesn't help the developer who's working with the library as it currently stands. Method documentation should describe how things *are*, issue trackers and defect lists should describe how things *could be*.